easybuild/easyconfigs/r/R/R-3.1.3-goolf-1.7.20-Java-1.7.0_80.eb

```diff
@@ -0,0 +1,386 @@
+name = 'R'
+version = '3.1.3'
+
+homepage = 'http://www.r-project.org/'
+description = """R is a free software environment for statistical computing and graphics."""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['http://cran.us.r-project.org/src/base/R-%(version_major)s']
+
+preconfigopts = 'BLAS_LIBS="$LIBBLAS" LAPACK_LIBS="$LIBLAPACK"'
+configopts = "--with-lapack --with-blas --with-pic --enable-threads --with-x=no --enable-R-shlib"
+# some recommended packages may fail in a parallel build (e.g. Matrix), and we're installing them anyway below
+configopts += " --with-recommended-packages=no"
+
+java = 'Java'
+javaver = '1.7.0_80'
+versionsuffix = '-%s-%s' % (java, javaver)
+dependencies = [
+    ('libreadline', '6.3'),
+    ('ncurses', '5.9'),
+    ('libpng', '1.6.16'),  # for plotting in R
```

Comment:
Your versions above are the same as in #1689 (and many recent PRs),
yet it would be nice if you could bump this just a bit towards `'libpng', '1.6.17'`;
this helps to have a bunch of sites cross-checking each others' builds; would that work for you?